### PR TITLE
Strike announcement Nov 29

### PR DESCRIPTION
**Pull request by wm75**
I think a height-reduced version of https://www.scientists4future.org/wp-content/uploads/2019/05/s4f-warming-stripes.png would be nice to have, too, but I was afraid to mess up the layout.

### CODE REVIEW

**Comment by anuprulez** [comment]
+1

File: _posts/2019-11-21-we-are-striking-really.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+site: freiburg
+title: All has been said - act!
```
Comment (simonbray) [inline _posts/2019-11-21-we-are-striking-really.md]:
```suggestion
title: Enough talking - time to act!
```

Comment (wm75) [reply _posts/2019-11-21-we-are-striking-really.md]: Well, the current title is the slogan of a recent scientists protest in Berlin (https://www.scientists4future.org/2019/10/es-wurde-alles-gesagt-handeln/) - that's why I chose it. But yours isn't bad either.

Comment (simonbray) [reply _posts/2019-11-21-we-are-striking-really.md]: Ah okay that makes sense. It just sounds a little strange translated directly into English in my view.

**Comment by stephanflemming** [comment]
Maybe add an invitation to join the Freiburg team on Friday?

**Pull request merged**